### PR TITLE
CVS-52640 Image transformation node (resize, layout, color order) 

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: example_client/east_ocr_client.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 import grpc
 import cv2
+import os
```
Comment (atobiszei) [inline example_client/east_ocr_client.py:19]: Left by mistake?

Comment (dkalinowski) [reply example_client/east_ocr_client.py:19]: Previous commit had it missing. So without this line currently develop branch has broken east ocr client.

File: src/custom_nodes/image_transformation/image_transformation.cpp [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+        (*info)[0].dims[1] = targetImageHeight == -1 ? 0 : targetImageHeight;
+        (*info)[0].dims[2] = targetImageWidth == -1 ? 0 : targetImageWidth;
+        (*info)[0].dims[3] = targetImageColorOrder == "GRAY" ? 1 : 3;
+    } else {
+        (*info)[0].dims[1] = targetImageColorOrder == "GRAY" ? 1 : 3;
+        (*info)[0].dims[2] = targetImageHeight == -1 ? 0 : targetImageHeight;
+        (*info)[0].dims[3] = targetImageWidth == -1 ? 0 : targetImageWidth;
+    }
+
+    (*info)[0].precision = FP32;  // TODO: ?
```
Comment (atobiszei) [inline src/custom_nodes/image_transformation/image_transformation.cpp:303]: TODO

**Review by atobiszei — APPROVED** [review]

File: src/custom_nodes/image_transformation/image_transformation.cpp (hunk 2 above)
Comment (dkalinowski) [reply src/custom_nodes/image_transformation/image_transformation.cpp]: Done

**Review by dtrawins — APPROVED** [review]

**Review by michalkulakowski — APPROVED** [review]

**Pull request merged**